src/vignette/http/legacy/route_helpers.clj

```diff
@@ -21,6 +23,22 @@
 (defn archive? [map]
   (= (:zone map) (str "/" archive-dir)))
 
+(defn browser-supports-webp? [request]
```

Comment:
@Grunny I was thinking about that but since legacy are in theory to be removed soon™️ I wanted this code to be self contained and easily removable